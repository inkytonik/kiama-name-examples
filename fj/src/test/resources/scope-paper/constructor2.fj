// class/constructors.spt:construct Object
new Object()
