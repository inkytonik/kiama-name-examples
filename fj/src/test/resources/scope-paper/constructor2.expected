src/test/resources/scope-paper/constructor2.fj:2:1:info: new Object ...: Object
new Object()
^
Object
